tests/asgi/test_testing_asgi.py

```diff
@@ -102,3 +103,31 @@ def class_meth(cls, scope, receive, send):
             pass
 
     assert testing.client._is_asgi_app(Foo.class_meth)
+
+
+def test_cookies_jar():
```

Comment:
This could probably be DRYied by employing the `asgi` fixture and the dual client, just as other tests do, but it is not a hard requirement from me to pass review.

Reply:
Did you mean, put the resource class into `asgi/_asgi_test_app` module?

Reply:
No, we are able to utilize existing sync tests under ASGI if certain envvars are set.
But I realize this is quite an obscure addition that virtually no one has picked up. We'll probably need to describe this in `CONTRIBUTING.md` or another piece of documentation meant for Falcon contributors.